fury/shaders/sdf_dec.vert

```diff
@@ -0,0 +1,7 @@
+/* SDF vertex shader declaration */
+
+//VTK::ValuePass::Dec
+
+in vec3 center;
+out vec3 centeredVertexMC;
+out vec3 centt;
```

Comment:
Should be named `centerWCVSOutput`. The centers defined by the user are in world coordinates, right?